spk/octoprint/Makefile

```diff
@@ -1,22 +1,25 @@
-SPK_NAME = octoprint
-SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 1
-SPK_ICON = src/octoprint.png
+SPK_NAME   = octoprint
+SPK_VERS   = $(shell date +%Y%m%d)-devel
```

Comment:
`-devel` doesn't belong here, please remove.

If you want to release a testing/develop/nightly version of Octoprint, then you may release a completely separate package as such (see nzbget-testing, sabnzbd-testing etc). In the main Octoprint package, we should use stable releases (or at least as stable as possible)